flusher.go

```diff
@@ -53,7 +53,9 @@ func Flush(postMetrics [][]DDMetric) {
 		[]string{"part:marshal"},
 		1.0,
 	)
-	Stats.Gauge("flush.content_length_bytes", float64(reqBody.Len()), nil, 1.0)
+	// Len reports the unread length, so we have to record this before it's POSTed
+	bodyLength := reqBody.Len()
+	Stats.Gauge("flush.content_length_bytes", float64(bodyLength), nil, 1.0)
```

Comment:
Normally this should be a histogram but in this case we only do it every 10s so it wouldn't do anything.